src/operators/validate_byte_range.cc

```diff
@@ -37,6 +37,11 @@ bool ValidateByteRange::getRange(const std::string &rangeRepresentation,
                 "' into a number");
             return false;
         }
+        if ((start < 0) || (start > 255)) {
```

Comment:
I think you need the same check after line 50, since the start of the range could also be invalid. The check of `end` together with the check of `start > end` will technically do the same, but the user will just see `Invalid range...`, instead of `Invalid range start value:...`.